pkg/kubelet/volumemanager/reconciler/reconstruct_common.go

```diff
@@ -37,6 +38,10 @@ import (
 	utilstrings "k8s.io/utils/strings"
 )
 
+var _ logr.Marshaler = podVolume{}
+var _ logr.Marshaler = reconstructedVolume{}
+var _ logr.Marshaler = globalVolumeInfo{}
```

Comment:
Please add  a comment that these interfaces are necessary to keep the structures private and at the same time log them correctly in structured logs.

Reply:
done